conda-recipe/meta.yaml

```diff
@@ -13,15 +13,12 @@ build:
   number: {{ GIT_DESCRIBE_NUMBER|int }}
   script_env:
     - SDC_CONFIG_PIPELINE_SDC
+    - WHEELS_OUTPUT_FOLDER
 
 requirements:
   build:
     - {{ compiler('c') }}      # [not osx]
     - {{ compiler('cxx') }}    # [not osx]
-    - wheel
-    - python
-    - numba   {{ NUMBA_VERSION }}
```

Comment:
Numba recently became a build dependency for sdc, so it's better to keep it.

Reply:
Ok, let's keep numba here